lib/cli.rb

```diff
@@ -57,11 +58,19 @@ class Cli
 
   GLI::Commands::Help.skips_post = false
 
-  def self.check_exceeding_arguments(defined, parsed)
-    if parsed.size > defined.size
+  def self.validate_number_of_arguments(defined, parsed)
+    multiple = defined.select {|argument| argument.options.
+      include?(:multiple) && !a.options.include?(:options) }.
```

Comment:
Expression at 63, 60 should be on its own line.<br>Align the operands of an expression in an assignment spanning multiple lines.